storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,277 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import {
+  Card,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  LinkList,
+  PageHeading,
+  Paragraph,
+  SearchField,
+  Spotlight,
+  UnorderedList,
+} from '@amsterdam/design-system-react'
+import {
+  CameraIcon,
+  ChatBubbleIcon,
+  FacebookIcon,
+  LinkedinIcon,
+  TwitterIcon,
+} from '@amsterdam/design-system-react-icons'
+import { Meta, StoryObj } from '@storybook/react'
+import { NavigationColumn } from './NavigationColumn'
+import { NavigationPage } from './NavigationPage'
+import { burgerzakenLinks, parkerenLinks } from './links'
+import { exampleHeading, exampleParagraph } from '../../../components/shared/exampleContent'
+import { commonMeta } from '../common/config'
+
+const meta = {
+  ...commonMeta,
+  title: 'Pages/Amsterdam.nl/Navigation Page',
+  component: NavigationPage,
+} satisfies Meta<typeof NavigationPage>
+
+export default meta
+
+export const Default: StoryObj = {
+  args: {
+    children: (
+      <Grid gapVertical="large" paddingBottom="large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <PageHeading className="ams-mb--md">Burgerzaken</PageHeading>
+          <Paragraph size="large">
+            Heeft u een paspoort, rijbewijs of uittreksel nodig? Of wilt u een verhuizing doorgeven of een geboorte
+            aangeven? Op deze pagina vindt u alle informatie en regelzaken rondom Burgerzaken.
+          </Paragraph>
+        </Grid.Cell>
+        <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+          <NavigationColumn sections={burgerzakenLinks[0]} />
+        </Grid.Cell>
+        <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }}>
+          <NavigationColumn sections={burgerzakenLinks[1]} />
+        </Grid.Cell>
+      </Grid>
+    ),
+  },
+}
+
+const parkerenLorem =
+  'Aenean lorem elit, commodo eget elit sit amet, auctor venenatis ligula. Phasellus faucibus fringilla viverra. Aliquam vestibulum vehicula enim, ac consequat nisl mattis id.'
+
+export const WithMap: StoryObj = {
+  args: {
+    children: (
+      <>
+        <Grid gapVertical="large" paddingBottom="large">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+            <PageHeading className="ams-mb--md">Parkeren</PageHeading>
+            <Paragraph className="ams-mb--md" size="large">
+              {parkerenLorem}
+            </Paragraph>
+            <Paragraph className="ams-mb--md">{parkerenLorem}</Paragraph>
+            <Paragraph>{parkerenLorem}</Paragraph>
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <NavigationColumn sections={parkerenLinks[0]} />
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }}>
+            <NavigationColumn sections={parkerenLinks[1]} />
+          </Grid.Cell>
+        </Grid>
+        <Spotlight>
```

Comment:
This Spotlight is a little tall on wider windows, maybe we can divide it in 2 columns as well?

Reply:
@alimpens We simplified the content here, @Bondt007 agrees that this serves as a good example. Relieves the need for 2 columns I think.